x-pack/filebeat/tests/system/test_http_endpoint.py

```diff
@@ -193,7 +184,7 @@ def test_http_endpoint_get_request(self):
         payload = {self.prefix: message}
         headers = {"Content-Type": "application/json", "Accept": "application/json"}
         r = requests.get(self.url, headers=headers, data=json.dumps(payload))
-
+        print("response:", r.status_code, r.text)
```

Comment:
I assume this is just a debug print which does not belong here. Do you mind removing it from master as well?

Reply:
it belongs here. nosetest supresses output by default, but if the test fails (or if you enable verbose mode) then everything you printed will be displayed in the context of the test. The 'log' is here to give you a hint why the assert did fail. using `assert` in python tests just tells you where it did fail, not why.